firefox-ios/Client/pings.yaml

```diff
@@ -114,3 +114,27 @@ onboarding-opt-out:
   metadata:
     follows_collection_enabled: false
     include_info_sections: false
+    
+usage-deletion-request:
+  description: |
+    This ping is submitted when a user opts out of sending usage
+    frequency of Firefox to Mozilla.
+    Sent in response to user action.
+  include_client_id: false
+  send_if_empty: true
+  bugs:
+    - https://mozilla-hub.atlassian.net/browse/FXIOS-10902
+  data_reviews:
+    - https://github.com/mozilla-mobile/firefox-ios/pull/24263
+  notification_emails:
+    - glean-team@mozilla.com
+    - jrediger@mozilla.com
+    - loines@mozilla.com
+  metadata:
+    follows_collection_enabled: false
+    include_info_sections: false
+  reasons:
+    set_upload_enabled: |
+      The ping was submitted between Glean init and Glean shutdown.
+      After init but before shutdown the upload of usage reporting data changed
+      from enabled to disabled.
```

Comment:
```suggestion
      The ping was submitted due to the usage-reporting data preference changing from
      enabled to disabled.
```

Reply:
I said that before on, I think, the desktop implementation: due to this being a `include_info_sections=false` ping we won't see any reasons anyway. They are stripped from the final payload.

Reply:
Note that desktop did land with the reason defined. So for documentation reasons we can probably keep it here too (it's not set in the above `submit` call anyway)